### ISSUE TITLE
Cannot login

### ISSUE THREAD

**Issue by skuro** [keybase/client#3743]
When I try to login, after I input my Keybase password I get a straight error message, stating that somehow the privkey is missing. Here's some relevant information:

```
$ keybase --version
keybase version 1.0.16-20160715110019+f0caa34
$ keybase login
Your keybase username or email address: carlo.sciolla@gmail.com
▶ ERROR Bad key found: no private key material or GPGKey
$ gpg -K
-------------------------------
sec   2048R/5A50F60B 2016-07-25
uid                  Carlo Sciolla <carlo.sciolla@gmail.com>
ssb   2048R/A9E158FA 2016-07-25
ssb   4096R/F3C580C1 2016-07-25
```

Am I missing anything?

my log id: 157b0f9f71aafbbb0df0f31c

**Comment by maxtaco** [comment]
Is it possible that your pgp key expired?

**Comment by skuro** [comment]
Nope, the key is still pretty valid:

```
$ gpg --edit-key carlo.sciolla@gmail.com
[...]
pub  2048R/5A50F60B  created: 2016-07-25  expires: never       usage: SC
                     trust: ultimate      validity: ultimate
sub  2048R/A9E158FA  created: 2016-07-25  expires: never       usage: E
sub  4096R/F3C580C1  created: 2016-07-25  expires: never       usage: S
[ultimate] (1). Carlo Sciolla <carlo.sciolla@gmail.com>
[ultimate] (2)  [jpeg image of size 5899]
```

**Comment by maxtaco** [comment]
Seems like the Go client is having trouble using your server-synced private key. Super hard to debug since obviously we can't see your unencrypted private key.  Maybe try removing your private key from server sync (using the Web site) and then trying again?  Thanks for your feedback!

**Comment by skuro** [comment]
As in following the "reset my keys & start from scratch" option?

**Comment by maxtaco** [comment]
No, click on the "edit" tooltip next to your GPG fingerprint, and then select "Delete my private key from keybase"  You can of course export if if you need a copy, but my guess is you don't.

**Comment by skuro** [comment]
Oh! I might see what's going on here!

**tl;dr:** my privkey is *really* not uploaded at keybase

The initial export of my privkey came from my work laptop, in which the master key was *removed* and only signing subkeys were available in the keyring, while now I'm on my private laptop where the keyring is complete (as you can see in my original `gpg -K` log the key is marked as `sec`, while on my work laptop it's marked as `sec#`).

This explains why there's no real privkey found by [the login process](https://github.com/keybase/client/blob/32595a82032349e971334fb498224e9b130da309/go/libkb/pgp_key.go#L499). 

I guess I really need to start from scratch, do I? :-)

Anyways, that's something sensible to do as a sanity check when someone uploads his privkey.

**Comment by maxtaco** [comment]
Cool, that was my hunch, but thanks for confirming it!  This issue hasn't come up very much (first time, actually!) but you're right, we should do some more checking in the upload process. Thanks for your feedback!

**Comment by skuro** [comment]
No problems, I very much love the idea behind Keybase, anything I can do to help is my pleasure ;-)

**Comment by skuro** [comment]
Hey hey, just to let you know that I restarted the process and it's fully working at the moment. Thanks for your support!
